Automattic/wp-calypso
# Stats: Add a toggleable mobile menu for list items

#### Proposed Changes

<img width="452" alt="image" src="https://user-images.githubusercontent.com/4044428/202542226-65bc5fb6-876a-4f92-820a-e079357ce672.png">

* Restores action buttons for stats module list items.

#### Testing Instructions



* Spin up this PR in a live branch.
* Navigate to Calypso Stats in a mobile-sized viewport (`/stats/day/` at window width of <480px).
* Ensure that you see an ellipsis button that spawns a mobile dropdown menu when pressed.

#### Pre-merge Checklist





Related to #70020 and #69044.
